ocaml/idl/schematest.ml

```diff
@@ -3,7 +3,7 @@ let hash x = Digest.string x |> Digest.to_hex
 (* BEWARE: if this changes, check that schema has been bumped accordingly in
    ocaml/idl/datamodel_common.ml, usually schema_minor_vsn *)
 
-let last_known_schema_hash = "ab570d662d657ee2055194110d536302"
+let last_known_schema_hash = "a3ddab734ad88468120269cf3d0e3bd9"
```

Comment:
You still need to update the schema version in ocaml/idl/datamodel_common.ml.

Reply:
Sorry I thought I had done this already. It might have been lost during one of my rebase. Updated now.